workflow-bot-app/src/main/java/com/symphony/bdk/workflow/monitoring/service/MonitoringService.java

```diff
@@ -131,12 +131,12 @@ public WorkflowDefinitionView getWorkflowDefinition(String workflowId) {
     return builder.build();
   }
 
-  public List<VariableView> listWorkflowInstanceGlobalVars(String workflowId, String instanceId, String occurredBefore,
-      String occurredAfter) {
+  public List<VariableView> listWorkflowInstanceGlobalVars(String workflowId, String instanceId, String updatedBefore,
+      String updatedAfter) {
     // check if the instance belongs to the provided workflow
     this.checkIsInstanceOfWorkflow(workflowId, instanceId);
 
-    return variableQueryRepository.findGlobalVarsHistoryByWorkflowInstId(instanceId, occurredBefore, occurredAfter)
+    return variableQueryRepository.findGlobalVarsHistoryByWorkflowInstId(instanceId, updatedBefore, updatedAfter)
```

Comment:
should be better to parse the string into Instant at controller level, so that for the rest of code, we use instant, only in repository we parse it to Date. I think we can directly use Instant type for the query params in the controller